src/docs/customize/theming.mdx

```diff
@@ -87,14 +85,21 @@ Theming options affecting all form components.
 | `--rui-form-field-required-sign-color`               | Color of text appended to required input labels              |
```

Comment:
It might be beyond the scope of this issue, but I think it would be helpful, if the custom property table would also include data type and example and default if applicable.

I guess I could look in the default theme, but that is not very practical.

Reply:
Thanks for pointing it out. To be honest, I'm still not convinced of maintaining the theming docs manually, it's a _lot_ of work. There is an idea of generating the docs for it automatically but it's really an early concept based on discussions of @bedrich-schindler and me.

https://github.com/react-ui-org/react-ui/issues/224